### PR TITLE
Remove FixedTechTopSlot switch

### PR DESCRIPTION
**Pull request by regiskuckaertz**
@guardian/commercial-dev

### CODE REVIEW

**Comment by JonNorman** [comment]
:+1:

File: common/app/views/support/Commercial.scala [hunk 1]
```diff
@@ -54,9 +49,7 @@ object Commercial {
     }
 
     def slotCssClasses(metaData: MetaData): Seq[String] = {
```
Comment (rich-nguyen) [inline common/app/views/support/Commercial.scala]: this could be a `val` now

File: common/app/views/support/Commercial.scala [hunk 2]
```diff
@@ -29,17 +29,12 @@ object Commercial {
...
-
     def adSizes(metaData: MetaData, edition: Edition, maybeTags: Option[Tags]): Map[String, Seq[String]] = {
       val fabricAdvertsTop = Seq("88,71")
       val fluidAdvertsTop = Seq("fluid")
-      val leaderboardAdvertsTop = if (FixedTechTopSlot.isSwitchedOn && isUKTechFront(metaData)) None else Some("728,90")
       Map(
-        "tablet" -> (Seq("1,1", "88,70") ++ leaderboardAdvertsTop ++ fabricAdvertsTop ++ fluidAdvertsTop),
-        "desktop" -> (Seq("1,1", "88,70") ++ leaderboardAdvertsTop ++ Seq("940,230", "900,250", "970,250") ++ fabricAdvertsTop ++ fluidAdvertsTop)
+        "tablet" -> (Seq("1,1", "88,70", "728,90") ++ fabricAdvertsTop ++ fluidAdvertsTop),
+        "desktop" -> (Seq("1,1", "88,70", "728,90") ++ Seq("940,230", "900,250", "970,250") ++ fabricAdvertsTop ++ fluidAdvertsTop)
```
Comment (rich-nguyen) [inline common/app/views/support/Commercial.scala]: `Seq("1,1", "88,70", "728,90", "940,230", "900,250", "970,250")`

**Comment by rich-nguyen** [comment]
👍

**Pull request merged**